content/post/shipit-from-anywhere-lessons-learned/index.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "ShipIt from Anywhere: Our Lessons on Going Virtual"
+authors: ["Whitney Roark"]
+date: 2020-12-03
+tags: [culture, engineering, hackathon, shipit]
+thumbnail: "shipit-shirts.png"
+---
+
+ShipIt Days are Cerner Engineering's 24-hour hackathons where associates are empowered to brainstorm, plan, build, and present a project of their choice. This quarterly event is something associates enjoy by taking a break from their normal routine and focusing on a problem that they seek to achieve within a short time period (one day). The enjoyment of these projects often come from just working on something different, working with other colleagues, and just applying their passions to an area they have been desiring to solve. With the pandemic this year, we had to rethink how can we continue to achieve this same associate enjoyment and fulfillment while only being in a virtual format. 
```

Comment:
Can we remove the word "just" before "applying their passions" it sounds a bit odd.

Reply:
Applied changes: 5ad6e07